run_example.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+if [ "$#" -ne 1 ]
+then
+    echo "Usage: $0 [android|ios]"
+    exit
+fi
+
+# Updates, builds, and loads the example app in an IDE.
+# All commands have been copied from the project's CircleCI workflow.
+
+# Update and build the root-level components..
+npm install
+npm run build
+npm install -g @ionic/cli
+
+pushd example
+
+# Update and build the example app.
+npm install
+ionic build
+
+# Open example app in a Xcode or Android Studio.
```

Comment:
Can you update comment here? This commands should not open the IDE, it should run the app. Also, the first line I believe is what is needed to ensure the capacitor library stuff is updated on the paths needed for the build. Finally, for Android, I believe this is missing the build step (see circleci config)

Reply:
I just ran it again--it just opens the project Android Studio. For iOS, it opens the project in Xcode.

Reply:
Maybe just drop the comment? For me, it runs on the command line and asks which device to deploy on, then runs the app on that device or emulator. Maybe it is dependent on the IDE? I have mine running basically all the time already.

Reply:
I tried your other suggestion of running `./gradlew assemble` from the `example` directory, but there's no `gradlew` file. At this point, since the script was just meant to be a quick way for me to build and run on iOS, and shouldn't hold up this merge, I'm tempted just to make it iOS-specific.

Reply:
@jrtibbetts looks like you removed it again? Yes, if you can, please just make this a `run_ios_example.sh` file instead. I'd prefer to just use Gradle for android anyway.

Reply:
Welp. I fixed it the other day, but had committed it to a different branch. I've now fixed it for both platforms. Even if you still use Gradle directly, I'd like to keep the Android support in the script for when I run and test it.

Reply:
I removed it from the git repo.